output/mapping/eforms/guidance.yaml

```
@@ -102,7 +102,9 @@
     eForms guidance: |-
         The values of `cbc:NoticeTypeCode[@listName]` are drawn from the [form type authority table](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/form-type) in the EU Vocabularies.
 
-        Look up `cbc:NoticeTypeCode[@listName]` in the [form type mapping table](codelists/form-type), add the values of the release tag column to `tag` and set `tender.status` to the value of the tender status column.
+        If the code is "bri" discard this notice, it details information not modelled by OCDS.
```

Reuse language from [what's not included](https://standard.open-contracting.org/profiles/eforms/latest/en/primer/#what-s-not-included) (notice types X01 and X02 are the only notices with `formTyp` "bri").

```suggestion
        If the code is "bri", discard the notice. It relates to events outside the lifecycle of a contracting process.
```